rasterio/rio/main.py

```diff
@@ -30,15 +30,19 @@
 so that other ``rio`` users may find it.
 """
 
-
+import itertools
 import logging
-from pkg_resources import iter_entry_points
 import sys
 
 from click_plugins import with_plugins
 import click
 import cligj
 
+if sys.version_info < (3, 10):
+    from importlib_metadata import entry_points
```

Comment:
peanut gallery here, doesn't this introduce a package dependency on importlib-metdata ?

I seemed to need to add one for conda-forge/rasterio-feedstock#296

Reply:
@akrherz it does, for Pythons 3.8 and 3.9. https://github.com/rasterio/rasterio/blob/1.3.10/setup.py#L270 is supposed to install importlib-metadata for those versions. Does that not work?

Reply:
Oh sorry, I only reviewed `pyproject.toml` and did not find it there.